plasmapy/formulary/tests/test_dielectric.py

```diff
@@ -206,7 +206,7 @@ def test_normal_vs_lite_values(self, kwargs, expected):  # noqa: ARG002
         `permittivity_1D_Maxwellian` calculate the same values.
         """
 
-        wp = plasma_frequency(kwargs["n"], kwargs["particle"], kwargs["z_mean"])
+        wp = plasma_frequency(kwargs["n"], kwargs["particle"], Z=kwargs["z_mean"])
```

Comment:
Is there a reason not to change `z_mean` -> `Z` in this kwarg dict? Not that it really matters in the test, but to be consistent...